packages/vue-dot/src/elements/DialogBox/DialogBox.vue

```diff
@@ -38,14 +38,14 @@
 						v-bind="options.cancelBtn"
 						@click="$emit('cancel')"
 					>
-						{{ locales.cancel }}
+						{{ btnCancelText?btnCancelText:locales.cancel }}
```

Comment:
```suggestion
						{{ btnCancelText }}
```
Pour mettre la valeur par défaut, on utilise la valeur par défaut de la propriété !